### ISSUE TITLE
Only one test class is run using mvn

### ISSUE THREAD

**Issue by AleksandrSl** [kotlintest/kotlintest#274]
When run from command line:
```
mvn test
```

Only tests from one test class are performed.
Example project, based on your mvn example - https://gitlab.com/AleksandrSl/kotlintest-sandbox/blob/master/src/test/kotlin/io/kotlintest/samples/maven/ThirdTest.kt

**Comment by sksamuel** [comment]
Thanks for the bug report.
I've fixed this and released 3.0.3. Can you please confirm.

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running SecondTest
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.01 sec <<< FAILURE! - in SecondTest
should equals 5  Time elapsed: 0.008 sec  <<< FAILURE!
java.lang.AssertionError: expected: 5 but was: 4

Running StringTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 sec - in StringTest
Running ThirdTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0 sec <<< FAILURE! - in ThirdTest
should always fail  Time elapsed: 0 sec  <<< ERROR!
java.lang.IllegalArgumentException


Results :

Failed tests: 
  SecondTest expected: 5 but was: 4
Tests in error: 
  ThirdTest » IllegalArgument

Tests run: 4, Failures: 1, Errors: 1, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15.311 s
[INFO] Finished at: 2018-04-05T00:29:58+01:00
[INFO] Final Memory: 37M/424M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project kotlintest-samples-maven: There are test failures.

```

**Comment by AleksandrSl** [comment]
All is working now, thanks!
